Gopkg.toml

```diff
@@ -34,7 +34,7 @@
 
 [[constraint]]
   name = "github.com/fnproject/fn_go"
-  version = "0.2.7"
+  version = "0.2.8"
```

Comment:
You should run a dep-ensure locally to update the lock file and check that in  (otherwise dep ensure --vendor-only craps itself)

Reply:
Instead of changing it manually?